app/assets/javascripts/channels/synchronized_editor_channel.js

```diff
@@ -18,11 +20,14 @@ $(document).on('turbolinks:load', function() {
 
       received(data) {
         // Called when there's incoming data on the websocket for this channel
-        console.log(data);
+        if (current_user_id !== data['current_user_id']) {
+          CodeOceanEditor.applyChanges(data['delta']['data']);
+        }
       },
 
-      speak(data) {
-        this.perform('speak', { message: data });
+      send_changes(delta) {
+        const delta_with_user_id = {current_user_id: current_user_id, delta: delta}
+        this.perform('send_changes', { delta_with_user_id: delta_with_user_id});
```

Comment:
We don't have a linter in JavaScript yet. Therefore, we should try to fix the code ourselves (or at least use IntelliJ's code formatting with <kbd>command</kbd> + <kbd>option</kbd> + <kbd>L</kbd>).